nepali/about_nepal.md

```diff
@@ -1,4 +1,4 @@
-नेपाल एक सुनदर देश हो । यहाँका पाकृितक समपदा 
```

Comment:
I haven't made a single change to this line. Why is it displaying difference? @casey-collab